docs/photoniq/prerendering/04-prerendering-strategies/01-prefetching/configure-prefetching.md

```diff
@@ -89,11 +88,11 @@ Follow these instructions to edit a prefetching in the web console.
 </TabItem>
 <TabItem value="api" label="REST API">
 
-Use our interactive API Reference with code generation in 18 programming languages to [get prefetching configuration](https://www.macrometa.com/docs/apiPrerendering#/paths/api-prerender-v1-origins-origin--prefetch-configs/get).
+[Get prefetching configuration](https://www.macrometa.com/docs/apiPrerendering#/paths/api-prerender-v1-origins-origin--prefetch-configs/get).
```

Comment:
Here should be Update prefetching configuration. Please also get the correct URL.